public/index.html

```diff
@@ -17,59 +18,62 @@ <h1 class="header__wordmark">
           fluent
         </h1>
         <nav class="header__nav">
-          <a class="link" href="http://projectfluent.io/fluent/guide">Syntax Guide</a>
-          <a class="link" href="http://projectfluent.io/play">Online Editor</a>
-          <a class="link" href="https://github.com/projectfluent">GitHub</a>
-          <a class="link" href="https://groups.google.com/forum/#!forum/mozilla.tools.l10n">Mailing list</a>
-          <a class="link" href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
+          <a class="link" href="http://projectfluent.io/fluent/guide">Learn</a>
+          <a class="link" href="http://projectfluent.io/play">Play</a>
+          <a class="link" href="#implementation-docs">Use</a>
+          <a class="link" href="https://github.com/projectfluent">Contribute</a>
+          <!-- <a class="link" href="https://groups.google.com/forum/#!forum/mozilla.tools.l10n">Community</a> -->
         </nav>
       </header>
 
       <section class="section section--centered">
-        <p class="pitch">
-
-          <strong class="highlight">Fluent</strong> is a localization paradigm
-          designed to unleash the expressive power of the natural language.
-
-        </p>
+        <div class="pitch-wrapper">
+          <p class="pitch">
+            <strong>Fluent</strong> is a set of localization libraries<br>
+            designed to create better translations.
+          </p>
+          <a class="link use" href="#implementation-docs">See how to use it.</a>
+        </div>
 
-        <a class="link--cta" href="https://github.com/projectfluent">Visit GitHub →</a>
+        <div class="cta-wrapper">
+          <a class="link--cta" href="https://www.npmjs.com/package/fluent">Install fluent.js 0.3.1</a>
+          <a class="link other" href="https://github.com/projectfluent/">Other implementations</a>
```

Comment:
I really don't like this CTA, sorry. Fluent is not just fluent.js.  And fluent.js is not just one package. It might be `fluent-dom` or `fluent-react` etc. I feel like any "simple" CTA is just blurring things instead of helping to clarify them.

Perhaps the "Implementation docs" should explain all the different varieties that Fluent comes in, and the CTA should be the link to the docs?